trio_websocket/_impl.py

```diff
@@ -94,18 +94,19 @@ def __exit__(self, ty, value, tb):
 
 @asynccontextmanager
 async def open_websocket(
-        host: str,
-        port: int,
-        resource: str,
-        *,
-        use_ssl: Union[bool, ssl.SSLContext],
-        subprotocols: Optional[Iterable[str]] = None,
-        extra_headers: Optional[list[tuple[bytes,bytes]]] = None,
-        message_queue_size: int = MESSAGE_QUEUE_SIZE,
-        max_message_size: int = MAX_MESSAGE_SIZE,
-        connect_timeout: float = CONN_TIMEOUT,
-        disconnect_timeout: float = CONN_TIMEOUT
-    ):
+    host: str,
+    port: int,
+    resource: str,
+    *,
+    use_ssl: Union[bool, ssl.SSLContext],
+    subprotocols: Optional[Iterable[str]] = None,
+    extra_headers: Optional[list[tuple[bytes,bytes]]] = None,
+    message_queue_size: int = MESSAGE_QUEUE_SIZE,
+    max_message_size: int = MAX_MESSAGE_SIZE,
+    receive_buffer_size: int = RECEIVE_BYTES,
+    connect_timeout: float = CONN_TIMEOUT,
+    disconnect_timeout: float = CONN_TIMEOUT
+):
```

Comment:
I'd prefer to avoid the reformatting in this PR

Reply:
Given the type hint PR did all the reformatting this no longer is reformatted (heh).